### PR TITLE
doc: make private location configuration pages more consistent between them, closes HYB-448

### PR DESCRIPTION
**Pull request by Obero**

### CODE REVIEW

File: content/reference/admin/private_locations/configuration/azure/index.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ lastmod: 2023-10-13T08:10:39+00:00
...
 ---
 
-## Azure Virtual Machines
+## Permissions
 
-Azure private locations require the control plane to have Azure credentials configured in order to instantiate virtual machines and associated resources.
+Azure private locations require the control plane to have credentials configured in order to instantiate virtual machines and associated resources.
 
-### Environment variables
+Those can be set through environment variables in your control plane or via [Azure RBAC](https://learn.microsoft.com/en-us/azure/role-based-access-control/overview). Select the most appropriated method depending on your infrastructure, or check our [installation guide]({{< ref "../../installation/azure" >}})  for deployment with [Azure Container Apps](https://azure.microsoft.com/en-us/products/container-apps).
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/azure/index.md:14]:
```suggestion
Credentials can be set through environment variables in your control plane or via [Azure RBAC](https://learn.microsoft.com/en-us/azure/role-based-access-control/overview). Select the most appropriate method depending on your infrastructure, or check our [installation guide]({{< ref "../../installation/azure" >}})  for deployment with [Azure Container Apps](https://azure.microsoft.com/en-us/products/container-apps).
```

File: content/reference/admin/private_locations/configuration/azure/index.md [hunk 2]
```diff
@@ -26,7 +26,22 @@ Check Azure documentation pages to find these values:
 * [Client id and secret](https://learn.microsoft.com/en-us/answers/questions/834401/hi-i-want-my-client-id-and-client-secret-key)
 * [Subscription id](https://learn.microsoft.com/en-us/azure/azure-portal/get-subscription-tenant-id)
 
-### Control plane configuration file
+## System requirements
+
+Azure private locations rely on some dependencies.
+
+So when using a custom image, make sure following are available:
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/azure/index.md:33]:
```suggestion 
When using a custom image, make sure the following are available:
```

File: content/reference/admin/private_locations/configuration/azure/index.md [hunk 3]
```diff
@@ -26,7 +26,22 @@ Check Azure documentation pages to find these values:
...
+
+- [cloud-init](https://learn.microsoft.com/en-us/azure/virtual-machines/custom-data) integration.
+- [jq](https://jqlang.github.io/jq/download/) a lightweight and flexible command-line JSON processor.
+- [curl](https://curl.se/download.html) a command line tool and library for transferring data with URLs
+- [Java runtime environment](https://openjdk.org/install/): OpenJDK 64bits LTS versions: 11, 17 or 21 (see [Gatling prerequisites](https://gatling.io/docs/gatling/tutorials/installation/#java-version))
+
+{{< alert tip >}}
+Learn how to tune the OS for more performance, configure the open files limit, the kernel and the network [here](https://gatling.io/docs/gatling/reference/current/core/operations/).
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/azure/index.md:41]:
```suggestion
Learn how to tune the OS for more performance and configure the open files limit, the kernel, and the network [here](https://gatling.io/docs/gatling/reference/current/core/operations/).
```

Comment (stb13579) [inline content/reference/admin/private_locations/configuration/azure/index.md:41]: I usually avoid using "here" for a link in the documentation. You could replace "here" with "in the Documentation" and add the link to the word Documentation.

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 4]
```diff
@@ -16,16 +16,23 @@ The control plane must have access to your dedicated machines. Ensure that each
...
 
-The location host environment must include:
 - `bash` the GNU Project's shell
 - [jq](https://jqlang.github.io/jq/download/) a lightweight and flexible command-line JSON processor.
 - [curl](https://curl.se/download.html) a command line tool and library for transferring data with URLs
 - [Java runtime environment](https://openjdk.org/install/): OpenJDK 64bits LTS versions: 11, 17 or 21 (see [Gatling prerequisites](https://gatling.io/docs/gatling/tutorials/installation/#java-version))
-- `~/.ssh/authorized_keys` with a control plane public key
+- `~/.ssh/authorized_keys` with control plane public key
+
+{{< alert tip >}}
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:31]: see Azure comments

File: content/reference/admin/private_locations/configuration/ec2/index.md [hunk 5]
```diff
@@ -7,13 +7,29 @@ lastmod: 2023-10-13T08:10:39+00:00
...
 AWS private locations require the control plane to have access to AWS credentials from the default credential provider chain.
 
 See [the AWS documentation for the Default Credential Provider Chain](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html#credentials-default).
 
-### Control plane configuration file
+## System requirements
+
+AWS EC2 private locations rely on some dependencies.
+
+So when using a custom image, make sure following are available:
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/ec2/index.md:20]:
```suggestion
So when using a custom image, make sure the following are available:
```

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 6]
```diff
@@ -16,16 +16,23 @@ The control plane must have access to your dedicated machines. Ensure that each
 
 The control plane possesses a private key for establishing connections, while the corresponding public key is shared and configured on every host in the location.
 
-### Host prerequisites
+## System requirements
+
+Dedicated Machines private locations rely on some dependencies.
+
+So, make sure following are available on the location configured hosts:
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:23]:
```suggestion
So, make sure the following are available on the location-configured hosts:
```

File: content/reference/admin/private_locations/configuration/ec2/index.md [hunk 7]
```diff
@@ -7,13 +7,29 @@ lastmod: 2023-10-13T08:10:39+00:00
...
+
+- [cloud-init](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html) integration.
+- [jq](https://jqlang.github.io/jq/download/) a lightweight and flexible command-line JSON processor.
+- [curl](https://curl.se/download.html) a command line tool and library for transferring data with URLs
+- [Java runtime environment](https://openjdk.org/install/): OpenJDK 64bits LTS versions: 11, 17 or 21 (see [Gatling prerequisites](https://gatling.io/docs/gatling/tutorials/installation/#java-version))
+
+{{< alert tip >}}
+Learn how to tune the OS for more performance, configure the open files limit, the kernel and the network [here](https://gatling.io/docs/gatling/reference/current/core/operations/).
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/ec2/index.md:28]: see Azure comments

**Review by stb13579 — APPROVED** [review]
Looks good to me. Just some minor suggested changes.

**Review by slandelle — APPROVED** [review]

**Pull request merged**